src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -503,8 +496,8 @@ public virtual bool IsGuest
                 if (_safeTokenHandle.IsInvalid)
                     return false;
 
-                return CheckNtTokenForSid(new SecurityIdentifier(IdentifierAuthority.NTAuthority,
-                                                new int[] { Interop.SecurityIdentifier.SECURITY_BUILTIN_DOMAIN_RID, (int)WindowsBuiltInRole.Guest }));
+                ReadOnlySpan<int> subAuthorities = stackalloc int[2] { Interop.SecurityIdentifier.SECURITY_BUILTIN_DOMAIN_RID, (int)WindowsBuiltInRole.Guest };
```

Comment:
You shouldn't need the number with an explicit initializer.

```suggestion
                ReadOnlySpan<int> subAuthorities = stackalloc int[] { Interop.SecurityIdentifier.SECURITY_BUILTIN_DOMAIN_RID, (int)WindowsBuiltInRole.Guest };
```

Reply:
Fixed all occurences.